development/language/guidelines.rst

```diff
@@ -0,0 +1,190 @@
+                       topic_notify.txt
+                       user_activate.txt
+                       user_activate_inactive.txt
+                       user_activate_passwd.txt
+                       user_reactivate_account.txt
+                       user_remind_inactive.txt
+                       user_resend_inactive.txt
+                       user_welcome.txt
+                       user_welcome_inactive.txt
+                    app.php
+                    AUTHORS (optional)
+                    captcha_qa.php
+                    captcha_recaptcha.php
+                    cli.php
+                    CHANGELOG (optional)
+                    common.php
+                    groups.php
+                    help_bbcode.php
+                    help_faq.php
+                    index.htm (optional)
+                    install.php
+                    iso.txt
+                    LICENSE
+                    mcp.php
+                    memberlist.php
+                    migrator.php
+                    plupload.php
+                    posting.php
+                    README (optional)
+                    search.php
+                    ucp.php
+                    VERSION (optional)
+                    viewforum.php
+                    viewtopic.php
+              styles/
+                 prosilver/
+                    theme/
+                       {iso}/
+                          icon_user_online.gif
+                          index.htm (optional)
+                          stylesheet.css
+                 subsilver2/
+                    theme/
+                       {iso}/
+                          button_pm_new.gif
+                          button_pm_reply.gif
+                          button_topic_locked.gif
+                          button_topic_new.gif
+                          button_topic_reply.gif
+                          icon_contact_aim.gif
+                          icon_contact_email.gif
+                          icon_contact_icq.gif
+                          icon_contact_jabber.gif
+                          icon_contact_msnm.gif
+                          icon_contact_pm.gif
+                          icon_contact_www.gif
+                          icon_contact_yahoo.gif
+                          icon_post_delete.gif
+                          icon_post_edit.gif
+                          icon_post_info.gif
+                          icon_post_quote.gif
+                          icon_post_report.gif
+                          icon_user_offline.gif
+                          icon_user_online.gif
+                          icon_user_profile.gif
+                          icon_user_search.gif
+                          icon_user_warn.gif
+                          index.htm (optional)
+                          stylesheet.css
+
+5) Submissions have to follow as much as possible the `3.1 Translation (i18n/L10n) Guidelines`_ recommendations, especially the `3.1 Writing style`_.
```

Comment:
`Submissions should follow the recommendations in the `3.1 Translation (i18n/L10n) Guidelines`_ as closely as possible, especially the `3.1 Writing Style`_.`